### PR TITLE
update ajax for supervisor add subject to course

### PR DESCRIPTION
**Pull request by at-uytran**
image

### CODE REVIEW

**Comment by at-uytran** [comment]
ready

File: app/views/supervisor/course_subjects/create.js.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$('#new_course_subject_<%= @subject.id %>').replaceWith('<%= j(render "supervisor/courses/button/remove_subject", object: @cou_subject) %>');
+
```
Comment (tranducquoc) [inline app/views/supervisor/course_subjects/create.js.erb]: remove

**Comment by tranducquoc** [comment]
commented

**Comment by at-uytran** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**